### ISSUE TITLE
Question on the output weights after training

### ISSUE THREAD

**Issue by lannguyen0910** [hysts/pytorch_mpiigaze#59]
Hi @hysts , thank you for your delicate work!

I have a question on the output weights after training on MPIIGaze dataset. Looks like the model saves the checkpoints for all 15 person ID. However, I'm not fully understand how to use the checkpoint. Is the last checkpoint in person 15 similar to the one in the [pretrained models](https://github.com/hysts/pytorch_mpiigaze_demo/releases/tag/v0.1.0). If it isn't, then how can i create the weight similar to the ```pretrained models``` one from 15 output ID?

I hope that you will give me some clarifications. Thank you so much in advance!

Hoang-Lan Nguyen

**Comment by hysts** [comment]
Hi, @lannguyen0910 

The pre-trained models provided in this repo are retrained with data from all 15 people.
The training scripts like [scripts/run_all_mpiigaze_lenet.sh](https://github.com/hysts/pytorch_mpiigaze/blob/master/scripts/run_all_mpiigaze_lenet.sh) are for the leave-one-person-out evaluation protocol, which is used in [the MPIIGaze paper](https://arxiv.org/abs/1504.02863). I used the scripts to see if this repo reproduces the results reported in the paper, and to find a good model and hyperparameters. After that, I retrained the model with the same hyperparameters with data from all 15 people to create the pretrained model.

**Comment by lannguyen0910** [comment]
> The pre-trained models provided in this repo are retrained with data from all 15 people.
The training scripts like [scripts/run_all_mpiigaze_lenet.sh](https://github.com/hysts/pytorch_mpiigaze/blob/master/scripts/run_all_mpiigaze_lenet.sh) are for the leave-one-person-out evaluation protocol, which is used in [the MPIIGaze paper](https://arxiv.org/abs/1504.02863). I used the scripts to see if this repo reproduces the results reported in the paper, and to find a good model and hyperparameters. After that, I retrained the model with the same hyperparameters with data from all 15 people to create the pretrained model.

Thank you so much for the clarification @hysts . I have one more question left.
Now when i want to train a new model, i think i should just run the script with the param ```test_id=-1``` right? Do i have to concern about modifying the code in any other modules? 
Thank you!

Hoang-Lan Nguyen
